src/scripts/cfo.zig

```diff
@@ -502,39 +502,39 @@ test "cfo: SeedRecord.merge" {
             \\    "commit_sha": "2222222222222222222222222222222222222222",
```

Comment:
Curious that the timestapm/seed order is correct in the input for the tests. I guess I coded correct order first, and then just sorted the fields out of a habit  without noticing :rofl: